packages/cardpay-cli/scheduled-payment/enable-module.ts

```diff
@@ -0,0 +1,43 @@
+import { Argv } from 'yargs';
+import { getConstant, getSDK } from '@cardstack/cardpay-sdk';
+import { getEthereumClients, getConnectionType, NETWORK_OPTION_ANY } from '../utils';
+import { Arguments, CommandModule } from 'yargs';
+
+export default {
+  command: 'enable-module <safeAddress> <gasTokenAddress>',
+  describe: 'Enable scheduled payment module on the safe',
+  builder(yargs: Argv) {
+    return yargs
+      .positional('safeAddress', {
+        type: 'string',
+        description: 'The address of the safe whose enables the scheduled payment module',
+      })
+      .positional('gasTokenAddress', {
+        type: 'string',
+        description: 'The token address (defaults to Kovan DAI)',
```

Comment:
If anything, this should default to Goerli DAI since we have decided to use goerli as our testnet. This can wait for a separate PR.